README.md

```diff
@@ -304,28 +314,28 @@ To build demo app for specific platform follow the below instructions.
-1. From the menu of the `Unity` app, select __File > Build Settings...__
-1. Select `Android` from the __Platform__ window
-1. If `Android` is not the active platform, hit the button that says __Switch Platform__ (it will only be available if you actually need to switch active platforms)
-1. Press the __Build And Run__ button
-1. You will be asked to provide a location to save the build. Make a new directory at *unity-acpcore/ACPCore/Builds* (this folder is in the .gitignore file)
-1. Name build whatever you want and press __Save__
-1. `Unity` will build an `apk` file and automatically deploy it to the connected device
+2. From the menu of the `Unity` app, select __File > Build Settings...__
+3. Select `Android` from the __Platform__ window
+4. If `Android` is not the active platform, hit the button that says __Switch Platform__ (it will only be available if you actually need to switch active platforms)
+5. Press the __Build And Run__ button
+6. You will be asked to provide a location to save the build. Make a new directory at *unity-acpcore/ACPCore/Builds* (this folder is in the .gitignore file)
+7. Name build whatever you want and press __Save__
+8. `Unity` will build an `apk` file and automatically deploy it to the connected device
 
 ###### iOS
 1. From the menu of the `Unity` app, select __File > Build Settings...__
-1. Select `iOS` from the __Platform__ window
-1. If `iOS` is not the active platform, hit the button that says __Switch Platform__ (it will only be available if you actually need to switch active platforms)
-1. Press the __Build And Run__ button
-1. You will be asked to provide a location to save the build. Make a new directory at *unity-acpcore/ACPCore/Builds* (this folder is in the .gitignore file)
-1. Name build whatever you want and press __Save__
-1. `Unity` will create and open an `Xcode` project
-1. From the Xcode project run the app on a simulator.
-1. If you get an error `Symbol not found: _OBJC_CLASS_$_WKWebView`. Select the Unity-iPhone target -> Go to Build Phases tab -> Add `Webkit.Framework` to `Link Binary with Libraries`.
-
-## Additional Cordova Plugins
-
-Below is a list of additional Unity plugins from the AEP SDK suite:
+2. Select `iOS` from the __Platform__ window
+3. If `iOS` is not the active platform, hit the button that says __Switch Platform__ (it will only be available if you actually need to switch active platforms)
+4. Press the __Build And Run__ button
+5. You will be asked to provide a location to save the build. Make a new directory at *unity-acpcore/ACPCore/Builds* (this folder is in the .gitignore file)
+6. Name build whatever you want and press __Save__
+7. `Unity` will create and open an `Xcode` project
+8. [Add XCFrameworks to the Xcode project](#ios-installation).
+9. From the Xcode project run the app on a simulator.
+
+## Additional Unity Plugins
+
+Below is a list of additional Unity plugins from the ACP SDK suite:
 
 | Extension | GitHub | Unity Package |
 |-----------|--------|-----|
```

Comment:
for consistency, can we rename the Extension values to: ACPAnalytics, AEPAssurance and also update the versions